docs/email/components.html

```diff
@@ -0,0 +1,528 @@
+---
+layout: page
```

Comment:
I would find it useful here to have the secondary navigation. There's a lot of stuff in here and it's not immediately clear what's on this page with a title like "Components".

Reply:
If you haven't already discovered, this page _does_ have the secondary navigation. Jekyll caches templates, so a refresh fixes it.